pgcli_sublime.py

```diff
@@ -115,8 +115,10 @@ def on_query_completions(self, view, prefix, locations):
         if not completer:
             return
 
-        text = get_entire_view_text(view)
-        cursor_pos = view.sel()[0].begin()
+        # Get current query
+        current_query = get_current_query(view)
+        text = current_query[0]
+        cursor_pos = current_query[1]
```

Comment:
I think this would read a lot better in one line: `text, cursor_pos = get_current_query(view)`